azurerm/resource_arm_stream_analytics_job.go

```diff
@@ -42,7 +42,7 @@ func resourceArmStreamAnalyticsJob() *schema.Resource {
 
 			"compatibility_level": {
 				Type:     schema.TypeString,
-				Required: true,
+				Optional: true,
```

Comment:
🤔 on reflection could we update the `TestAccAzureRMStreamAnalyticsJob_basic` test to skip these properties (and add a `TestAccAzureRMStreamAnalyticsJob_complete` test with these set - to ensure that we're verifying both use-cases work?

Reply:
Agree with this sentiment

Reply:
Done